pkg/apis/serving/v1alpha1/domainmapping_types.go

```diff
@@ -82,6 +90,13 @@ type DomainMappingSpec struct {
 	// This contract is satisfied by Knative types such as Knative Services and
 	// Knative Routes, and by Kubernetes Services.
 	Ref duckv1.KReference `json:"ref"`
+
+	// TLS indicates the existing or expected tls secret that should be used for certificate generation.
+	//
+	// If defined it will use the existing tls secret with the given name, otherwise it will attempt to create
+	// a new secret with the expected name to be used by the certificate.
```

Comment:
Is this still true ? As in will a new secret be created if one doesnt exist ?

Reply:
Might also want to update the description in the PR and original issue if the requirements have changed as we explicitly don't want to create a secret if it doesn't exist